articles/azure-functions/durable-functions-disaster-recovery-geo-distribution.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Disaster Recovery and Geo Distribution in Durable Functions - Azure
+description: Learn about Disaster Recovery and Geo Distribution in Durable Functions.
+services: functions
+author: casard
+manager: cfowler
+editor: ''
+tags: ''
+keywords:
+ms.service: functions
+ms.devlang: multiple
+ms.topic: article
+ms.tgt_pltfrm: multiple
+ms.workload: na
+ms.date: 04/25/2018
+ms.author: azfuncdf
+---
+
+# Disaster Recovery and Geo-Distribution
+
+## Overview
+In Azure Durable Functions, all state is persisted in Azure Storage. A [task hub](durable-functions-task-hubs.md) is a logical container for Azure Storage resources that are used for orchestrations. Orchestrator and activity functions can only interact with each other when they belong to the same task hub.
+The scenarios described in this article propose deployment options to increase availability and minimize downtime during disaster recovery activities.
+It is important to notice that these scenarios are based on an Active-Passive configurations, since they are guided by the usage of Azure Storage.
+
+
+>[!NOTE]
+>All function apps that are redundantly deployed, must share the same function access keys in the case of being activated via HTTP. The Functions Runtime exposes a [management API](https://github.com/Azure/azure-functions-host/wiki/Key-management-API) that enables consumers to programmatically add, delete, and update function keys.
```

Comment:
Why is this note here?  Its relation to the overview, and the reason for formatting it as a note, is not clear.

Link to the docs.ms.com article for the management API rather than the wiki.

Reply:
We kept the note, so that we put emphasis on the importance of the scenario basics.

@tdykstra where can we find it in docs?

Reply:
Sorry, my mistake, I thought I remembered seeing such a doc but can't find it now.

@ggailey777 Do you know if we cover the material in https://github.com/Azure/azure-functions-host/wiki/Key-management-API anywhere in our docs?